calcom/cal.com
# fix: (managed-event-booking): slug conflict handling

## What does this PR do?



- Fixes #16363 
- Fixes CAL-4173

## ISSUE DEMO

https://github.com/user-attachments/assets/130f717d-1218-4cf2-8b23-4e5abc130666




## FIX DEMO
https://github.com/user-attachments/assets/ee8299b2-c580-4168-972e-381c36e405fd




## Mandatory Tasks (DO NOT REMOVE)

- [x] I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com/docs). N/A

## How should this be tested?


Steps to test:

- As a team admin create a new managed event type
- Create a personal user event for the team admin or any other user with the same slug as the managed event type
- In the assignment tab, assign a user ( for which you created event in last step) to the managed event type
- This should open the `ManagedEventDialog`. Earlier it was throwing `An unexpected Error occurred. Try again`.
## Checklist
